docs/index.rst

```diff
@@ -8,14 +10,50 @@ The *fledge-north-kafka* plugin sends data from Fledge to the an Apache Kafka. F
 
 The configuration of the *Kafka* plugin is very simple, consisting of four parameters that must be set.
 
-+---------+
-| |kafka| |
-+---------+
++-----------+
+| |kafka_1| |
++-----------+
 
   - **Bootstrap Brokers**: A comma separate list of Kafka brokers to use to establish a connection to the Kafka system.
 
-  - **Kafka Topic**: The Kafka topic to which all data is sent.
+  - **Topic**: The Kafka topic to which all data is sent.
 
   - **Send JSON**: This controls how JSON data points should be sent to Kafka. These may be sent as strings or as JSON objects.
 
   - **Data Source**: Which Fledge data to send to Kafka; Readings or Fledge Statistics.
+
++-----------+
+| |kafka_2| |
++-----------+
+
+  - **Security Protocol**: Security protocol to be used to connect with kafka broker.
+
+  - **SASL Mechanism**: SASL Mechanism. Only PLAIN mechanism is supported, support for other will be added soon
+
+  - **User ID**: User ID to be used with SASL_PLAINTEXT and SASL_SSL.
+
+  - **Password**: Password to be used with SASL_PLAINTEXT and SASL_SSL.
+
++-----------+
+| |kafka_3| |
++-----------+
+
+  - **Root CA Name**: Name of Root CA to use in certificate verification.
```

Comment:
Define what CA means, "Name of the root certificate authority that will be used to verify the certificate."

Reply:
Fixed